README.md

```diff
@@ -1 +1,57 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+
+> My First Microverse Project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo 
+
+[Coming Soon]
+
+
+## Getting Started
+
+**Go to your Cli and follow the steps below**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Clone the repository: go to Your Cli and run "git clone <repository url>"
+
+### change into the repository:  run 'cd <repository name>'
+
+### Install all the project dependences: Run 'npm Insttall' in your cli
+
+### now you can view the project in the browser using any web server of choice
```

Comment:
**[OPTIONAL]:** The #, ## and ### are normally used in Markdown to differentiatie headers, subheaders or separate sections by their title, because of this, in this case I think it could be more accurate :100: to use bullet points or a numeric list to detail each step.

If you want to learn more about using markdown for your README's and PR's here is [a link that offers some tips](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) :bulb: